pkgs/top-level/all-packages.nix

```diff
@@ -14603,6 +14603,12 @@ in
 
   pybind11 = pythonPackages.pybind11;
 
+  pylode = callPackage ../misc/pylode {
+    inherit (python3Packages)
+      buildPythonApplication dateutil falcon gunicorn isodate jinja2 markdown
+      rdflib rdflib-jsonld requests six beautifulsoup4;
```

Comment:
Could you please put all of those details inside `pylode/default.nix`? It's just not common to do this and it's not necessary - you can import `python3Packages` inside `pylode/default.nix`.

Reply:
Ah sorry, saw another packages doing it. Just corrected it. Thanks!